mne/viz/_brain/_brain.py

```diff
@@ -1493,6 +1493,10 @@ def add_data(self, array, fmin=None, fmid=None, fmax=None,
                                  f'{time_label_size}')
 
         hemi = self._check_hemi(hemi, extras=['vol'])
+        try:
+            stc, array, vertices = self._check_stc(hemi, array, vertices)
+        except ValueError:
+            return  # No data to add, skipping
```

Comment:
This seems weird -- someone calls `add_data` but no data gets added? Shouldn't this be an error? What is the use case?